zap/src/irgen/mod.rs

```diff
@@ -441,6 +442,15 @@ impl Display for Expr {
 			Self::True => write!(f, "true"),
 			Self::Nil => write!(f, "nil"),
 
+			Self::StrOrBool(string) => {
+				if string == "false" || string == "False" {
+					return write!(f, "false")
+				} else if string == "true" || string == "True" {
```

Comment:
See above - for the first option:
```suggestion
				} else if string == "true" {
```